addons/l10n_sa_edi/models/account_edi_xml_ubl_21_zatca.py

```diff
@@ -429,8 +429,11 @@ def _get_tax_unece_codes(self, invoice, tax):
         def _exemption_reason(code, reason):
             return {
                 'tax_category_code': code,
-                'tax_exemption_reason_code': reason,
-                'tax_exemption_reason': exemption_codes[reason].split(reason)[1].lstrip(),
+                'tax_exemption_reason_code': reason or "Not subject to VAT",
```

Comment:
Should not this be VATEX-SA-OOS @mebe-odoo ?

Reply:
@jco-odoo absolutely, I updated it